string mode
string default_mode
string spot_meter
string timelapse_interval
string auto_off
uint32 fov
string picres
uint32 mins_elapsed
uint32 secs_elapsed
string orientation
string charging
uint32 pics_remaining
uint32 n_pics
uint32 mins_remaining
uint32 n_vids
string record
string low_light
string protune
string white_balance
string looping
uint32 batt2
string overheated
string attachment
string vidres
uint32 fps


